README.md

```diff
@@ -3,9 +3,10 @@ cgm-remote-monitor (a.k.a. Nightscout)
 
 [![Build Status][build-img]][build-url]
 [![Dependency Status][dependency-img]][dependency-url]
+[![Coverage Status][coverage-img]][coverage-url]
 [![Gitter chat][gitter-img]][gitter-url]
-[![Stories in Ready][ready-img]][ready-url]
-[![Stories in Progress][progress-img]][progress-url]
+[![Stories in Ready][ready-img]][waffle]
```

Comment:
An unrelated local cleanup that I'm happy to punt to another PR (`ready-url` and `progress-url` are the same).